src/zdaemon/tests/tests.py

```diff
@@ -79,7 +79,7 @@ def dont_hang_when_program_doesnt_start():
     >>> write('conf',
     ... '''
     ... <runner>
-    ...   program sleep
+    ...   program sleep .1
```

Comment:
On Python 3.4 the zdaemon daemon terminates because it hits the "restarting too frequently" case in `governor()`.  I wonder why that doesn't happen on 3.5?